### ISSUE TITLE
prevent 'external-frontend' service to be created with Istio deployment version 

### ISSUE THREAD

**Issue by yogeek** [GoogleCloudPlatform/microservices-demo#251]
Hi !

First thank you for this great demo app.
I deploy this app automatically with a script and when deploying the Istio version with :
```
export MICRODEMO_NAMESPACE="microdemo"

# Enable Istio injection in namespace
kubectl label namespace ${MICRODEMO_NAMESPACE} istio-injection=enabled --overwrite

# Deploy microservices
kubectl apply -f ./release/istio-manifests.yaml -n ${MICRODEMO_NAMESPACE}
kubectl apply -f ./release/kubernetes-manifests.yaml -n ${MICRODEMO_NAMESPACE}
```

the frontend is exposed both with the 'frontend-external' LoadBalancer service (like in the 'without-istio' version) and with the Istio ingresss-gateway (with the 'frontend-gateway').

Is there a way to prevent deploying the 'frontend-external' service when deploying with Istio (to only have the Istio endpoint) in an automated way ? By this, I mean by deleting 'frontend-external' service from the  `./release/kubernetes-manifests.yaml` manifest with a kubectl command ?

Thank you

**Comment by ahmetb** [comment]
You can just delete that part of the YAML?
This is needed because this app is used without Istio as well.

**Comment by yogeek** [comment]
@ahmetb yes but the question was about how to delete it automatically (not manually, because my installation script download the release and install the application by using kubectl on the appropriate files). And as the manifests are all in the same yaml, it is not so easy to delete the sevice part in the script.
But I just found I can manage it by using kustomize with the `$patch: delete` since the following issue (https://github.com/kubernetes-sigs/kustomize/issues/768) is now closed.
Thank you for your answer !
